tesseract_urdf/include/tesseract_urdf/box.h

```diff
@@ -41,12 +41,14 @@ class XMLDocument;
 
 namespace tesseract_urdf
 {
+extern const char* BOX_ELEMENT_NAME;
```

Comment:
@Levi-Armstrong this seems to be a problem for windows. Do you know how to handle this for windows? If I don't make it `extern`, then I get the compiler `unused-variable` error

Reply:
Why are these needed?

Reply:
I would recommend create a single header like constants.h and leveraging `constexpr std::string_view BOX_ELEMENT_NAME = "box";`

Reply:
> Why are these needed?

I thought it would be better to save the URDF element names into variables rather than typing the strings multiple times where they get used